### ISSUE TITLE
BT-754, BT-755: Accessibility criteria are included in lot

### ISSUE THREAD

**Issue by odscjen** [open-contracting/european-union-support#126]
BT-754: Accessibility `cac:ProcurementAdditionalType​/cbc:ProcurementTypeCode`
BT-755: Accessiblity Justification `cac:ProcurementAdditionalType​/cbc:ProcurementType`

This pair take the form of a code from the [accessibility] (https://op.europa.eu/en/web/eu-vocabularies/concept-scheme/-/resource?uri=http://publications.europa.eu/resource/authority/accessibility) codelist and an accompanying description field to be used if no accessibility criteria are included but the procurement is intended for natural persons. The 3 codes are:
* inc - Accessibility criteria for persons with disabilities are included
* n-inc - Accessibility criteria for persons with disabilities are not included because the procurement is not intended for use by natural persons
* n-inc-just - Accessibility criteria for persons with disabilities are not included with the following justification

Suggest we model a new pair of objects to cover this, a `tender.lots.hasAccessibilityCriteria` boolean with 'inc' = "true" and 'n-inc', -n-inc-just' = "false". And an accompanying `tender.lots.noAccessibilityCriteriaRationale` free-text string field for the BT-755 justification.

**Comment by jpmckinney** [comment]
Sounds fine to me. Would we set `noAccessibilityCriteriaRationale` to "Procurement is not intended for use by natural persons" when it's 'n-inc'?

**Comment by odscjen** [comment]
yes that makes a lot of sense, the [guidance](https://docs.ted.europa.eu/eforms/1.3.2/schema/all-in-one.html#accessibilitySection) suggests that BT-755 will only be used if `ProcurementTypeCode` = 'n-inc-just' so `noAccessibilityCriteriaRationale` will be available for 'n-inc'

**Comment by odscjen** [comment]
I've updated the guidance so I'll close this issue now
